.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -10,24 +10,35 @@ assignees: ''
 
 ## Is your feature request related to a problem? Please describe
 
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+<!--
+Please provide a clear and concise description of what the problem is.
+E.g. I'm always frustrated when [...]
+-->
 
 ## How would you classify your suggestion
 
+<!--
```

Comment:
What's the benefit of having them on a line before and after the comment?

Easier to read?

Reply:
Seems to flow better